src/libraries/Common/tests/System/Security/Cryptography/X509Certificates/CertificateAuthority.cs

```diff
@@ -793,5 +832,131 @@ private enum CertStatus
             OK,
             Revoked,
         }
+
+        internal static void BuildPrivatePki(
+        PkiOptions pkiOptions,
+        out RevocationResponder responder,
+        out CertificateAuthority rootAuthority,
+        out CertificateAuthority intermediateAuthority,
+        out X509Certificate2 endEntityCert,
+        string testName = null,
+        bool registerAuthorities = true,
+        bool pkiOptionsInSubject = false,
+        string subjectName = null)
```

Comment:
Looks like all of the parameters got outdented when this method got moved.

```suggestion
            PkiOptions pkiOptions,
            out RevocationResponder responder,
            out CertificateAuthority rootAuthority,
            out CertificateAuthority intermediateAuthority,
            out X509Certificate2 endEntityCert,
            string testName = null,
            bool registerAuthorities = true,
            bool pkiOptionsInSubject = false,
            string subjectName = null)
```

Reply:
fixed. probably some VS magic I missed during copy.